src/actions/index.js

```diff
@@ -11,8 +11,16 @@ export const DECREMENT = 'DECREMENT';
 
 export const increment = () => {
   // Fill in this function
+  return {
+    type: "INCREMENT",
```

Comment:
```suggestion
    type: INCREMENT,
```
So we want to refer to the variable binding we created above, rather than the string here. This will help prevent hard to find bugs. When we give the compiler a misspelled variable, it will usually give us some sort of error or warning along the lines of 'undeclared variable' . If we just misspell a string the compiler doesn't recognize that as a problem and will run, but the program won't behave as expected. Then we have to find that bug ourselves, which gets harder as the program gets bigger.